peekaboo/toolbox/known.py

```diff
@@ -37,37 +38,73 @@ def __init__(self, sample, db_con):
         self.sample = sample
         self.db_con = db_con
 
-    def query_database_journal_for_sample(self):
-        """ Query database and return if this sample has
-        been analysed. """
-        return self.db_con.sample_info_fetch(self.sample)
-
     def get_report(self):
         """ Return knowntools report or create if not already cached. """
         if self.sample.knowntools_report is not None:
             return self.sample.knowntools_report
 
-        ktreport = KnowntoolsReport(self.query_database_journal_for_sample())
+        ktreport = KnowntoolsReport(
```

Comment:
Here'e you're defactoring a method you've introduced just a couple of commit earlier. Should we squash that?